src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private int index;
+
     public String getRandomColor() {
-        return null;
+        Color [] arrayColors = Color.values();
+        index = new Random().nextInt(arrayColors.length);
```

Comment:
Previous comment not fixed.
![image](https://user-images.githubusercontent.com/20326858/212892757-b57659ed-1d13-4517-8a8f-2484916e5c55.png)

Reply:
the variable index was announced inside the Color Supplier class.Is it not enough, for make a variable the class variable?

Reply:
not index variable

move new Random to the class variables and use that variable in your index calculation